README.md

```diff
@@ -1,2 +1,133 @@
-# carbonite-migrate
-Migrate devices using Carbonite Migrate.
+carbonite-migrate
+=================
```

Comment:
Update the Word document with any changes made here.

Reply:
Done

Reply:
Done. Reuploaded the doc file to the documentation ticket.